kong/tools/utils.lua

```diff
@@ -608,14 +608,16 @@ end
 -- @return success A boolean indicating wether the module was found.
 -- @return module The retrieved module, or the error in case of a failure
 function _M.load_module_if_exists(module_name)
-  local status, res = pcall(require, module_name)
+  local status, res = xpcall(require, function(err)
+                                        return debug.traceback(err)
+                                      end, module_name)
   if status then
     return true, res
   -- Here we match any character because if a module has a dash '-' in its name, we would need to escape it.
   elseif type(res) == "string" and find(res, "module '" .. module_name .. "' not found", nil, true) then
     return false, res
   else
-    error(res)
+    error(tostring(res).." while loading module '" .. tostring(module_name) .. "'")
```

Comment:
Nice! The name of the module that failed is probably the most useful piece of info, so I suggest bringing it up front:

```suggestion
    error("error loading module '" .. tostring(module_name) .. "':\n" .. tostring(res))
```
(I was actually going to suggest _adding_ the module name, because I missed it at the bottom of the stacktrace in the PR description)